tests/test_mr.py

```diff
@@ -366,3 +367,130 @@ def test_destroy_dm_mr(self):
                         dm_mr = DMMR(pd, dm_mr_len, e.IBV_ACCESS_ZERO_BASED,
                                      dm=dm, offset=dm_mr_offset)
                         dm_mr.close()
+
```

Comment:
Some general comments:
- In case there is no dri card those tests will fail, and they can't. you should not run/skip those tests in such cases (you can check it's done in other places by try-except and raising unittest.SkipTest(..) )
- It's preferred not to use random value. we don't have/maintain a random seed, so in case of failures because of some specific value it may not be possible for the user to reproduce it. If you see that it's important for some value to be randomized and has an added value for the test(s), then at least print those attrs and their values in case of failure (so the user can reproduce it manually / have an idea of what's happening)